deployments/helm-chart/templates/rbac.yaml

```diff
@@ -3,6 +3,7 @@ kind: ClusterRole
 apiVersion: rbac.authorization.k8s.io/v1beta1
 metadata:
   name: {{ include "nginx-ingress.name" . }}
+  namespace: {{ .Release.Namespace }}
```

Comment:
this will not cause errors, but `ClusterRole` is not scoped to a namespace. I suggest  removing it before merging.

Reply:
Yeah I wasn't sure that we needed it here, removing it 🙆‍♂ Thanks!